tests/framework/requirements/requirements.go

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
There is an existing package that does something similar `tests/framework/checks`, please consider adding the new logic there.

Reply:
The intention of this package is to have requirements for tests in a way if requirements doesnt meet tests should fail, such as the multi-node example.

The checks file seem to  have some functions for querying the state and doesnt do the actual failing.
I think the `requirements` package is a better fit, maybe some checks can be changed and moved to `requirements` later on.

Reply:
Could you please explain how does the proposed `MultipleNodes` function is different from the existing `checks.HasAtLeastTwoNodes`?

https://github.com/kubevirt/kubevirt/blob/5c0c407f8a61a2e2b73834a5f161d675408e89d4/tests/framework/checks/checks.go#L91

Currently it is being used for programmatic skips:
https://github.com/kubevirt/kubevirt/blob/70df69d90ca6396fd1daceb1709372381f9d3ba3/tests/canary_upgrade_test.go#L62-L64

I think it could also be used to check for prerequisites.

Reply:
This commit is removed, existing skip remain as is following review comments